File: flowershop-manager-api/FlowershopAPI/FlowershopAPI/Contracts/Sales/SaleResponses.cs
namespace FlowershopAPI.Contracts.Sales;

public class SaleResponse
{
    public Guid Id { get; set; }
    public string ProductName { get; set; } = string.Empty;
    public string WarehouseName { get; set; } = string.Empty;
    public string DestinationName { get; set; } = string.Empty;
    public int Quantity { get; set; }
    public decimal PriceAtSale { get; set; }
    public DateTimeOffset SaleDate { get; set; }
}

File: flowershop-manager-frontend/src/types/dtos/sales/saleResponses.ts
export interface SaleResponse {
  id: string,
  productName: string,
  warehouseName: string,
  destinationName: string,
  quantity: number,
  priceAtSale: number,
  saleDate: string,
}

export interface SaleResponseForEdit {
  id: string,
  productName: string,
  productId: string,
  warehouseName: string,
  destinationId: string,
  quantity: number,
  priceAtSale: number,
  saleDate: string,
}